packages/app-mobile/android/app/src/main/java/net/cozic/joplin/MainApplication.java

```diff
@@ -44,6 +45,7 @@ public boolean getUseDeveloperSupport() {
           // Packages that cannot be autolinked yet can be added manually here, for example:
           packages.add(new SharePackage());
           packages.add(new SslPackage());
+          packages.add(new TextInputPackage());
```

Comment:
When we do this, does it mean it overrides the TextInput component?

Reply:
it overrides a class that manages interactions between native EditText and React Native TextInput